docs/index.rst

```diff
@@ -4,68 +4,131 @@
+Zarr is a format for the storage of chunked, compressed, N-dimensional arrays
+inspired by `HDF5 <https://www.hdfgroup.org/HDF5/>`_, `h5py
+<https://www.h5py.org/>`_ and `bcolz <https://bcolz.readthedocs.io/>`_.
+
+The project is fiscally sponsored by `NumFOCUS <https://numfocus.org/>`_, a US
+501(c)(3) public charity, and development is supported by the
+`MRC Centre for Genomics and Global Health <https://www.cggh.org>`_
+and the `Chan Zuckerberg Initiative <https://chanzuckerberg.com/>`_.
+
+These documents describe the Zarr Python implementation. More information
+about the Zarr format can be found on the `main website <https://zarr.dev>`_.
 
 Highlights
 ----------
 
-* Create N-dimensional arrays with any NumPy dtype.
-* Chunk arrays along any dimension.
-* Compress and/or filter chunks using any NumCodecs_ codec.
-* Store arrays in memory, on disk, inside a Zip file, on S3, ...
-* Read an array concurrently from multiple threads or processes.
-* Write to an array concurrently from multiple threads or processes.
-* Organize arrays into hierarchies via groups.
+ * Create N-dimensional arrays with any NumPy dtype.
+ * Chunk arrays along any dimension.
+ * Compress and/or filter chunks using any NumCodecs_ codec.
+ * Store arrays in memory, on disk, inside a Zip file, on S3, ...
+ * Read an array concurrently from multiple threads or processes.
+ * Write to an array concurrently from multiple threads or processes.
+ * Organize arrays into hierarchies via groups.
 
-Status
-------
+Contributing
+------------
 
-Zarr is still a young project. Feedback and bug reports are very welcome, please get in touch via
+Feedback and bug reports are very welcome, please get in touch via
 the `GitHub issue tracker <https://github.com/zarr-developers/zarr-python/issues>`_. See
 :doc:`contributing` for further information about contributing to Zarr.
 
-Contents
---------
-
-.. toctree::
-    :maxdepth: 2
-
-    installation
-    tutorial
-    api
-    spec
-    release
-    contributing
-
 Projects using Zarr
 -------------------
 
 If you are using Zarr, we would `love to hear about it
-<https://github.com/zarr-developers/zarr-python/issues/228>`_.
+<https://github.com/zarr-developers/community/issues/19>`_.
 
 Acknowledgments
 ---------------
 
 The following people have contributed to the development of Zarr by contributing code,
 documentation, code reviews, comments and/or ideas:
 
-* :user:`Francesc Alted <FrancescAlted>`
-* :user:`Martin Durant <martindurant>`
-* :user:`Stephan Hoyer <shoyer>`
-* :user:`John Kirkham <jakirkham>`
-* :user:`Alistair Miles <alimanfoo>`
-* :user:`Mamy Ratsimbazafy <mratsim>`
-* :user:`Matthew Rocklin <mrocklin>`
-* :user:`Vincent Schut <vincentschut>`
-* :user:`Anthony Scopatz <scopatz>`
-* :user:`Prakhar Goel <newt0311>`
-
-Zarr is inspired by `HDF5 <https://www.hdfgroup.org/HDF5/>`_, `h5py
-<https://www.h5py.org/>`_ and `bcolz <https://bcolz.readthedocs.io/>`_.
+:user:`Alistair Miles <alimanfoo>`
```

Comment:
Wonder if we can put this in a place where it can be used here and in the changelog

Reply:
You mean like a file to `:include:`? I was wondering if we might not could put the `:user:` names _only here (and make a big :tada: for new contributors) rather than needing to list them on the release notes. (To make each PR a little bit easier)